maintenance/recipes/turn_on.rb

```diff
@@ -0,0 +1,6 @@
+bash 'turn maintenance on' do
+  estimated_end_time = node[:maintenance][:estimated_end_time]
+  sed_command = "s/<span id=\"estimated-end-time\">.*</span>/#{estimated_end_time}/"
+  code "sudo -S -u deploy sed -i \"#{sed_command}\" /srv/www/jiffyshirts/current/public/maintenance.html"
```

Comment:
```suggestion
bash 'turn maintenance on' do
  estimated_end_time = node[:maintenance][:estimated_end_time]
  sed_command = "s/<span id=\"estimated-end-time\">.*</span>/#{estimated_end_time}/"
  
  user 'deploy'
  code "sed -i \"#{sed_command}\" /srv/www/jiffyshirts/current/public/maintenance.html"
```

Reply:
Anyway, `template` resource is a chef-way of doing that: https://docs.chef.io/resources/template/#template-requirements

Reply:
Updated

Reply:
Sorry, that was an incorrect link, that is a correct one: https://docs.chef.io/resources/template/#syntax
I'd prefer using `template` instead of `execute` with `sed` command.